### ISSUE TITLE
Is page refreshing possible?

### ISSUE THREAD

**Issue by lostinblue** [skratchdot/open-golang#11]
I'm curious if I could refresh the page that is loaded.

**Comment by xiegeo** [comment]
If you are opening a http url, you can make the browser think it is a new page by adding a different query at the end. like: `http://example.com/?foo=1234` where `1234 `can be `i++` or `time.Now()`.

A longer answer is that open should open a new page anyway. If that page is not fresh, then probably the server has set a cache control header to not refresh for a certain time period.

**Comment by skratchdot** [comment]
I agree with the comment @xiegeo made above, but your question did get me thinking.  The only way I know of "refreshing" a page, is by using something like `live-reload`- but that's only for pages you own/control.  I don't think there's a cross platform solution to the problem you are talking about (mostly due to security concerns).

I did see someone talk about a way to do it on OSX for safari: https://apple.stackexchange.com/questions/189153/open-or-reload-html-doc-from-command-line

The problem is that I don't believe browsers expose information about which pages are open (for good reason)- and they don't AFAIK allow you to change the url of a random tab/window with something you provide.

In theory, there might be ways to do what you are trying to do, but they would involve spawing a server, using something like web sockets on the page you are trying to load/reload, as well as modifying the logic you use when calling `open.Run`.  So I could try to give you some ideas on how to hack around the problem you are trying to solve, but I would need more info on your use case / project, and it's also a problem that won't / shouldn't be solved by this library (although maybe a wrapper library that you create would be useful for people)...

Long story short, it's best to just follow xiegeo's advice, and deal with the multiple windows (unless you really want to invest some time into coming up w/ a better solution).

**Comment by xiegeo** [comment]
@skratchdot You got me thinking outside the box too.

@lostinblue If you can inject a bit of javascript to the page then you can also try this:
http://stackoverflow.com/questions/11313434/how-to-refresh-page-after-focus
